foursight_core/scripts/decrypt_accounts_file.py

```diff
@@ -0,0 +1,52 @@
+import io
```

Comment:
These scripts are for locally/manually encrypting/decrypting the accounts.json file (encrypted since it contains internal URLs). Only used currently for 4dn-dcic and only for different stages of this. Decrypted on the fly using the  ENCODED_AUTH0_SECRET value in the GAC.